src/github/client/github-installation-client.ts

```diff
@@ -39,19 +38,18 @@ import { GitHubClient } from "./github-client";
  * @see https://docs.github.com/en/developers/apps/building-github-apps/authenticating-with-github-apps
  */
 export class GitHubInstallationClient extends GitHubClient {
-	private readonly appTokenHolder: AppTokenHolder;
 	private readonly installationTokenCache: InstallationTokenCache;
 	public readonly githubInstallationId: InstallationId;
 	public readonly gitHubServerAppId?: number;
 
 	constructor(
 		githubInstallationId: InstallationId,
+		githubConfig: GitHubConfig,
```

Comment:
Nit: should have uppercase H in github. I know there's a lot of places in the code where that's not the case (need to clean that up) but should strive to avoid in new code.

Reply:
Well spotted! Thanks. Fixed :)